tests/core/test_model_manager.py

```diff
@@ -419,27 +392,26 @@ def test_fall_back_to_local(reset_globals):
     """Make sure that if LOCAL is enabled and a given module doesn't have any
     registered backends, the default caikit.core.load is used.
     """
-    with temp_config(
-        {
-            "module_backends": {
-                "priority": [],
-            }
-        }
-    ):
-        configure()
-        with temp_saved_model(NonDistributedBlock()) as model_path:
-            model = caikit.core.load(model_path)
+    with temp_saved_model(NonDistributedBlock()) as model_path:
+        model = caikit.core.load(model_path)
 
-        assert isinstance(model, NonDistributedBlock)
+    assert isinstance(model, NonDistributedBlock)
 
 
 def test_no_local_if_disabled(reset_globals):
     """Make sure that if LOCAL is disabled and a given module doesn't have any
     registered backends, loading fails.
     """
     _ = setup_saved_model(MockBackend)
+    # 🌶️ TODO: remove the `disable_local` flag
```

Comment:
I think the goal is to eventually remove all "special" handling for `LOCAL` and just have the default values in `config.yml` pre-populated with `LOCAL`. Could certainly argue either way, though

Reply:
right, needs more context.

The idea was to get `aconfig` to handle lists so that we could set `LOCAL` in the base config and allow users to merge more priorities to the list ahead of `LOCAL`, instead of handling it separately in code here